src/dtos/monitor-span.dto.ts

```diff
@@ -152,7 +153,7 @@ export class MonitorSpanBaseDTO {
       });
     },
   })
-  @Min(MPF_MIN_VALUE, {
+  @IfIsAtive((value) => min(value, MPF_MIN_VALUE), {}, {
```

Comment:
Similar to the comments on `monitor-default.dto.ts`, this will still fail for inactive records with an `mpfValue` less than the minimum because of the `@IsInRange` decorator (really, there's no need for `@Min` when there is `@IsInRange` is already applied). I suggest wrapping the `IsInRange` validation in `IfIsActive`, too.